cmd/karavictl/cmd/storage_create.go

```diff
@@ -191,41 +209,115 @@ var storageCreateCmd = &cobra.Command{
 		// TODO(ian): This logic should ideally be performed remotely, not
 		// in the client.
 
-		sioClient, err := goscaleio.NewClientWithArgs(epURL.String(), "", true, false)
-		if err != nil {
-			errAndExit(err)
-		}
+		var tempStorage SystemType
 
-		_, err = sioClient.Authenticate(&goscaleio.ConfigConnect{
-			Username: input.User,
-			Password: input.Password,
-		})
-		if err != nil {
-			errAndExit(err)
-		}
+		switch input.Type {
+		case powerflex:
+			tempStorage = storage[powerflex]
+			if tempStorage == nil {
+				tempStorage = make(map[string]System)
+			}
 
-		resp, err := sioClient.FindSystem(input.SystemID, "", "")
-		if err != nil {
-			errAndExit(err)
-		}
-		if resp.System.ID != input.SystemID {
-			fmt.Fprintf(cmd.ErrOrStderr(), "system id %q not found", input.SystemID)
-			osExit(1)
+			fmt.Printf("Connecting to PowerFlex system...\n")
```

Comment:
For consistency, should we also add output for "Connecting to Unisphere..."?  Or perhaps we should be less chatty and remove this type of output altogether.

Reply:
Decided to remove the powerflex output instead of adding more output. I think it looks cleaner.